tasks/utils/artifact.js

```diff
@@ -58,38 +61,38 @@ class RequestStore {
 
   async create(data /*: RequestData*/) /*: Promise<void> */ {
     validateDataVersion0(data)
-    if (await this.exists(data.requestId)) {
-      throw new Error(`Request ${data.requestId} already exists on chain ${this.chainId}`)
+    if (await this.exists(data.id)) {
+      throw new Error(`Request ${data.id} already exists on chain ${this.chainId}`)
```

Comment:
Do we want to throw an error here or log a warning?  Also, since the user could call `setAutoRequest` multiple times for the same contract to update the request, will this cause an unintended error?

Reply:
Kept it as throwing and added `.upsert` method for automation requests